diff --git a/app/Http/Controllers/HomeController.php b/app/Http/Controllers/HomeController.php
index 2da1d476c0dbb12b1c065cc9af026330885090a0..37f8eea80630289e03c4a665233c6c2025f2e7b1 100644
--- a/app/Http/Controllers/HomeController.php
+++ b/app/Http/Controllers/HomeController.php
@@ -4,13 +4,12 @@ namespace App\Http\Controllers;
 
 defined('DS') or exit('No direct script access allowed.');
 
-use App\Http\Controllers\Controller;
-use View, DB;
+use View;
 
 class HomeController extends Controller
 {
- public function index()
- {
-   return View::make('welcome');
- }
-}
\ No newline at end of file
+    public function index()
+    {
+        return View::make('welcome');
+    }
+}
diff --git a/resources/lang/id/faker/state_abbreviation.php b/resources/lang/id/faker/state_abbreviation.php
index bc15b3c6800cc65bfac7a4bdd57d619560b133da..794fa7b5326522061e8f2e7109eba39f35dac894 100644
--- a/resources/lang/id/faker/state_abbreviation.php
+++ b/resources/lang/id/faker/state_abbreviation.php
@@ -4,8 +4,8 @@ defined('DS') or exit('No direct script access allowed.');
 
 return [
     'Aceh', 'SumUt', 'SumBar', 'Jambi', 'BaBel', 'Riau', 'KepR', 'Bengkulu', 'SumSel',
-   'Lampung', 'Banten', 'DKI', 'JaBar', 'JaTeng', 'JaTim', 'NTT', 'DIY',
-   'Bali', 'NTB', 'KalBar', 'KalTeng', 'KalSel', 'KalTim', 'KalUt', 'SulSel',
-   'SulUt', 'Gorontalo', 'SulTeng', 'SulBar', 'SulTra', 'Maluku', 'MalUt',
-   'PapBar', 'Papua',
+    'Lampung', 'Banten', 'DKI', 'JaBar', 'JaTeng', 'JaTim', 'NTT', 'DIY',
+    'Bali', 'NTB', 'KalBar', 'KalTeng', 'KalSel', 'KalTim', 'KalUt', 'SulSel',
+    'SulUt', 'Gorontalo', 'SulTeng', 'SulBar', 'SulTra', 'Maluku', 'MalUt',
+    'PapBar', 'Papua',
 ];
diff --git a/system/Core/Router.php b/system/Core/Router.php
index 26e6d48924510dd92dc9c5756ef926b74128355d..ebe631eb5d98c5ae666635e8426ea76b2a6a64d7 100644
--- a/system/Core/Router.php
+++ b/system/Core/Router.php
@@ -85,7 +85,7 @@ class Router
     }
 
     /**
-     * Ambil list data grup yang terdaftar
+     * Ambil list data grup yang terdaftar.
      *
      * @return array
      */
@@ -160,9 +160,9 @@ class Router
     }
 
     /**
-     * Ambil satu atau seluruh middleware
+     * Ambil satu atau seluruh middleware.
      *
-     * @param  string $name
+     * @param string $name
      *
      * @return string
      */
@@ -180,9 +180,9 @@ class Router
     }
 
     /**
-     * Cek apakah middleware sudah terdaftar atau belum
+     * Cek apakah middleware sudah terdaftar atau belum.
      *
-     * @param  string $name
+     * @param string $name
      *
      * @return bool
      */
diff --git a/system/Database/Database.php b/system/Database/Database.php
index ed30c2fca2aafed4daf99d105f835f64aabb013e..9cdb366a458b5e2bfd658116432d908cf6f664ad 100644
--- a/system/Database/Database.php
+++ b/system/Database/Database.php
@@ -40,9 +40,9 @@ class Database
     }
 
     /**
-     * Ambil objek koneksi PDO dari sebuah koneksi database
+     * Ambil objek koneksi PDO dari sebuah koneksi database.
      *
-     * @param  string $connection
+     * @param string $connection
      *
      * @return \PDO
      */
diff --git a/system/Support/Str.php b/system/Support/Str.php
index 116d324f0f902e87d4cec6e936eae71e98f4dac8..0389f581b2a54b0afeec7d2578433822477efe5c 100644
--- a/system/Support/Str.php
+++ b/system/Support/Str.php
@@ -723,38 +723,38 @@ class Str
             'zh'   => ['ж', 'ჟ', 'ژ'],
             '(c)'  => ['©'],
             'A'    => ['Á', 'À', 'Ả', 'Ã', 'Ạ', 'Ă', 'Ắ', 'Ằ', 'Ẳ', 'Ẵ', 'Ặ', 'Â',
-                        'Ấ', 'Ầ', 'Ẩ', 'Ẫ', 'Ậ', 'Å', 'Ā', 'Ą', 'Α', 'Ά', 'Ἀ', 'Ἁ',
-                        'Ἂ', 'Ἃ', 'Ἄ', 'Ἅ', 'Ἆ', 'Ἇ', 'ᾈ', 'ᾉ', 'ᾊ', 'ᾋ', 'ᾌ', 'ᾍ',
-                        'ᾎ', 'ᾏ', 'Ᾰ', 'Ᾱ', 'Ὰ', 'Ά', 'ᾼ', 'А', 'Ǻ', 'Ǎ', 'Ａ', 'Ä', ],
+                'Ấ', 'Ầ', 'Ẩ', 'Ẫ', 'Ậ', 'Å', 'Ā', 'Ą', 'Α', 'Ά', 'Ἀ', 'Ἁ',
+                'Ἂ', 'Ἃ', 'Ἄ', 'Ἅ', 'Ἆ', 'Ἇ', 'ᾈ', 'ᾉ', 'ᾊ', 'ᾋ', 'ᾌ', 'ᾍ',
+                'ᾎ', 'ᾏ', 'Ᾰ', 'Ᾱ', 'Ὰ', 'Ά', 'ᾼ', 'А', 'Ǻ', 'Ǎ', 'Ａ', 'Ä', ],
             'B' => ['Б', 'Β', 'ब', 'Ｂ'],
             'C' => ['Ç', 'Ć', 'Č', 'Ĉ', 'Ċ', 'Ｃ'],
             'D' => ['Ď', 'Ð', 'Đ', 'Ɖ', 'Ɗ', 'Ƌ', 'ᴅ', 'ᴆ', 'Д', 'Δ', 'Ｄ'],
             'E' => ['É', 'È', 'Ẻ', 'Ẽ', 'Ẹ', 'Ê', 'Ế', 'Ề', 'Ể', 'Ễ', 'Ệ', 'Ë',
-                        'Ē', 'Ę', 'Ě', 'Ĕ', 'Ė', 'Ε', 'Έ', 'Ἐ', 'Ἑ', 'Ἒ', 'Ἓ', 'Ἔ',
-                        'Ἕ', 'Έ', 'Ὲ', 'Е', 'Ё', 'Э', 'Є', 'Ə', 'Ｅ', ],
+                'Ē', 'Ę', 'Ě', 'Ĕ', 'Ė', 'Ε', 'Έ', 'Ἐ', 'Ἑ', 'Ἒ', 'Ἓ', 'Ἔ',
+                'Ἕ', 'Έ', 'Ὲ', 'Е', 'Ё', 'Э', 'Є', 'Ə', 'Ｅ', ],
             'F' => ['Ф', 'Φ', 'Ｆ'],
             'G' => ['Ğ', 'Ġ', 'Ģ', 'Г', 'Ґ', 'Γ', 'Ｇ'],
             'H' => ['Η', 'Ή', 'Ħ', 'Ｈ'],
             'I' => ['Í', 'Ì', 'Ỉ', 'Ĩ', 'Ị', 'Î', 'Ï', 'Ī', 'Ĭ', 'Į',
-                        'İ', 'Ι', 'Ί', 'Ϊ', 'Ἰ', 'Ἱ', 'Ἳ', 'Ἴ', 'Ἵ', 'Ἶ', 'Ἷ',
-                        'Ῐ', 'Ῑ', 'Ὶ', 'Ί', 'И', 'І', 'Ї', 'Ǐ', 'ϒ', 'Ｉ', ],
+                'İ', 'Ι', 'Ί', 'Ϊ', 'Ἰ', 'Ἱ', 'Ἳ', 'Ἴ', 'Ἵ', 'Ἶ', 'Ἷ',
+                'Ῐ', 'Ῑ', 'Ὶ', 'Ί', 'И', 'І', 'Ї', 'Ǐ', 'ϒ', 'Ｉ', ],
             'J' => ['Ｊ'],
             'K' => ['К', 'Κ', 'Ｋ'],
             'L' => ['Ĺ', 'Ł', 'Л', 'Λ', 'Ļ', 'Ľ', 'Ŀ', 'ल', 'Ｌ'],
             'M' => ['М', 'Μ', 'Ｍ'],
             'N' => ['Ń', 'Ñ', 'Ň', 'Ņ', 'Ŋ', 'Н', 'Ν', 'Ｎ'],
             'O' => ['Ó', 'Ò', 'Ỏ', 'Õ', 'Ọ', 'Ô', 'Ố', 'Ồ', 'Ổ',
-                        'Ỗ', 'Ộ', 'Ơ', 'Ớ', 'Ờ', 'Ở', 'Ỡ', 'Ợ', 'Ø', 'Ō',
-                        'Ő', 'Ŏ', 'Ο', 'Ό', 'Ὀ', 'Ὁ', 'Ὂ', 'Ὃ', 'Ὄ', 'Ὅ',
-                        'Ὸ', 'Ό', 'О', 'Ө', 'Ǒ', 'Ǿ', 'Ｏ', 'Ö', ],
+                'Ỗ', 'Ộ', 'Ơ', 'Ớ', 'Ờ', 'Ở', 'Ỡ', 'Ợ', 'Ø', 'Ō',
+                'Ő', 'Ŏ', 'Ο', 'Ό', 'Ὀ', 'Ὁ', 'Ὂ', 'Ὃ', 'Ὄ', 'Ὅ',
+                'Ὸ', 'Ό', 'О', 'Ө', 'Ǒ', 'Ǿ', 'Ｏ', 'Ö', ],
             'P' => ['П', 'Π', 'Ｐ'],
             'Q' => ['Ｑ'],
             'R' => ['Ř', 'Ŕ', 'Р', 'Ρ', 'Ŗ', 'Ｒ'],
             'S' => ['Ş', 'Ŝ', 'Ș', 'Š', 'Ś', 'С', 'Σ', 'Ｓ'],
             'T' => ['Ť', 'Ţ', 'Ŧ', 'Ț', 'Т', 'Τ', 'Ｔ'],
             'U' => ['Ú', 'Ù', 'Ủ', 'Ũ', 'Ụ', 'Ư', 'Ứ', 'Ừ', 'Ử', 'Ữ',
-                        'Ự', 'Û', 'Ū', 'Ů', 'Ű', 'Ŭ', 'Ų', 'У', 'Ǔ', 'Ǖ',
-                        'Ǘ', 'Ǚ', 'Ǜ', 'Ｕ', 'Ў', 'Ü', ],
+                'Ự', 'Û', 'Ū', 'Ů', 'Ű', 'Ŭ', 'Ų', 'У', 'Ǔ', 'Ǖ',
+                'Ǘ', 'Ǚ', 'Ǜ', 'Ｕ', 'Ў', 'Ü', ],
             'V'    => ['В', 'Ｖ'],
             'W'    => ['Ω', 'Ώ', 'Ŵ', 'Ｗ'],
             'X'    => ['Χ', 'Ξ', 'Ｘ'],
diff --git a/tests/system/core/ControllerTest.php b/tests/system/core/ControllerTest.php
index 95f0541e92e0fbe630709510ad2fecbd319ce28c..5b917b63b6db1e724ef1beb4f5cbdd666a239547 100644
--- a/tests/system/core/ControllerTest.php
+++ b/tests/system/core/ControllerTest.php
@@ -22,7 +22,7 @@ class ControllerTest extends TestCase
         $this->assertNull($this->controller->middleware('auth'));
 
         $name = 'a-non-existant-middleware';
-        $exception =  '\InvalidArgumentException';
+        $exception = '\InvalidArgumentException';
 
         // fails
         if (PHP_VERSION_ID <= 50500) {
diff --git a/tests/system/core/RouterTest.php b/tests/system/core/RouterTest.php
index c1fa14b65b6f335e7396ebd02a77e3b2c7ee89e1..1fd835a0216bd96bc99ef88292b1649cd0d37b40 100644
--- a/tests/system/core/RouterTest.php
+++ b/tests/system/core/RouterTest.php
@@ -1,13 +1,12 @@
 <?php
 
 use PHPUnit\Framework\TestCase;
-use System\Debugger\Debugger;
 use System\Core\Router as Route;
+use System\Debugger\Debugger;
 
 class RouterTest extends TestCase
 {
-   
-   public function __construct()
+    public function __construct()
     {
         parent::__construct();
         require_once dirname(dirname(dirname(__DIR__))).'/index.php';
@@ -35,12 +34,12 @@ class RouterTest extends TestCase
     public function testMiddleware()
     {
         Route::middleware('auth');
-        
+
         $this->assertSame(
             Route::getMiddleware('auth'),
             'App\Http\Middleware\Authenticate@handle'
         );
-        
+
         $this->assertTrue(Route::hasMiddleware('auth'));
         $this->assertSame(Route::getMiddleware('foobar'), null);
     }
